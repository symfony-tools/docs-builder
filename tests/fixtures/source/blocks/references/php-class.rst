
:phpclass:`ArrayAccess`

:phpclass:`BcMath\Number`
