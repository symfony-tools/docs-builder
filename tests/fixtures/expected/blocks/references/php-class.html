<p><a href="https://secure.php.net/manual/en/class.arrayaccess.php" class="reference external" title="ArrayAccess" rel="external noopener noreferrer" target="_blank">ArrayAccess</a></p>
<p><a href="https://secure.php.net/manual/en/class.bcmath-number.php" class="reference external" title="BcMath\Number" rel="external noopener noreferrer" target="_blank">BcMath\Number</a></p>
